app/src/main/java/com/eveningoutpost/dexdrip/Models/TransmitterData.java

```diff
@@ -17,16 +19,16 @@
 
 @Table(name = "TransmitterData", id = BaseColumns._ID)
 public class TransmitterData extends Model {
-    private final static String TAG = BgReading.class.getSimpleName();
+    private final static String TAG = TransmitterData.class.getSimpleName();
 
     @Column(name = "timestamp", index = true)
     public long timestamp;
 
     @Column(name = "raw_data")
     public double raw_data;
-//
-//    @Column(name = "filtered_data")
-//    public double filtered_data;
+
+    @Column(name = "filtered_data")
```

Comment:
this will cause anyone attempting to update their app to crash on all new data unless it includes a migration, its already included in the other dexbridge branch